app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/TrackListScreen.kt

```diff
@@ -37,15 +42,21 @@ fun TrackListScreen(
           viewModel.setSearchQuery(query)
         },
         label = { Text("Search Tracks") },
-        modifier =
-            Modifier.fillMaxWidth()
-                .padding(16.dp)
-                .testTag("searchBar") // Adding a test tag for the search bar
-        )
+        modifier = Modifier.fillMaxWidth().padding(16.dp).testTag("searchBar"))
+
+    Row(verticalAlignment = Alignment.CenterVertically, modifier = Modifier.padding(16.dp)) {
+      Text("Show Recently Added")
```

Comment:
Don't use hard-coded text values. Create a resource string and reference it from here (also creates a sonarcloud issue)